src/components/preview-ghost/PreviewGhost.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from 'react';
-import Ghost from '../../../../../components/ghost/Ghost';
+import Ghost from '../ghost/Ghost';
 
 function PreviewGhost() {
```

Comment:
Can we rename this to `MessagePreviewGhost` and move it to a non-common folder to make it clear that it's not used in preview itself?

Reply:
This change will require to update some integration tests, but other than that I'm fine with it. Any thoughts @DanDeMicco?